doc/algorithms_config/map_configs.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
As you mentioned in the meeting, all tables have some columns whose titles are in portuguese.

Reply:
fixed, tnks

Reply:
Checked!